.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
pre-commit and link are not tied together.
We should use pre-commit as a subset of the lint checks we do in github actions.

We should add flake8 to the project (independent of pre-commit) and merge that as a separate commit

Reply:
After than, another commit for add pre-commit 
And document in this file maybe that the pre-commit configurations for black have not been added because of py3.6 compat issues between pre-commit and black